libraries/chain/proposal_evaluator.cpp

```diff
@@ -21,19 +21,134 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
+#include <graphene/chain/hardfork.hpp>
 #include <graphene/chain/proposal_evaluator.hpp>
 #include <graphene/chain/proposal_object.hpp>
 #include <graphene/chain/account_object.hpp>
+#include <graphene/chain/protocol/account.hpp>
 #include <graphene/chain/protocol/fee_schedule.hpp>
+#include <graphene/chain/protocol/tournament.hpp>
 #include <graphene/chain/exceptions.hpp>
+#include <graphene/chain/hardfork.hpp>
 
 #include <fc/smart_ref_impl.hpp>
 
 namespace graphene { namespace chain {
 
+struct proposal_operation_hardfork_visitor
+{
+   typedef void result_type;
+   const fc::time_point_sec block_time;
+
+   proposal_operation_hardfork_visitor( const fc::time_point_sec bt ) : block_time(bt) {}
+
+   template<typename T>
+   void operator()(const T &v) const {}
+
+   void operator()(const committee_member_update_global_parameters_operation &op) const {
+      if( block_time < HARDFORK_1000_TIME ) // TODO: remove after hf
+         FC_ASSERT( !op.new_parameters.extensions.value.min_bet_multiplier.valid()
+                    && !op.new_parameters.extensions.value.max_bet_multiplier.valid()
+                    && !op.new_parameters.extensions.value.betting_rake_fee_percentage.valid()
+                    && !op.new_parameters.extensions.value.permitted_betting_odds_increments.valid()
+                    && !op.new_parameters.extensions.value.live_betting_delay_time.valid(),
+                    "Parameter extensions are not allowed yet!" );
+   }
```

Comment:
Must prevent fees for new operations from being set before hardfork.

Reply:
This is something where we need assistance from you or @oxarbitrage.